WeakAurasOptions/AuthorOptions.lua

```diff
@@ -2528,7 +2558,7 @@ function OptionsPrivate.GetAuthorOptions(data)
           local i = #child.authorOptions + 1
           child.authorOptions[i] = {
             type = "toggle",
-            key = "option" .. i,
+            key = generateKey("option", child.authorOptions, i),
```

Comment:
small behavior change, the first option you create will have default key `option` instead of `option1`, then it'll go `option1`, option2`, etc. But I don't care about that change, nobody is ever expected to keep those keys in the first place